src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -140,6 +140,11 @@ public class MicrobeEditorGUI : Node
 
     private bool inEditorTab = false;
 
+    public LineEdit GetNewSpeciesName()
```

Comment:
This should return a string so that the caller doesn't have to know that this comes from a text box.